src/command_modules/azure-cli-cosmosdb/azure/cli/command_modules/cosmosdb/commands.py

```diff
@@ -30,6 +29,10 @@ def load_command_table(self, _):
         g.custom_command('update', 'cli_cosmosdb_update')
         g.custom_command('list', 'cli_cosmosdb_list')
 
+    # # key operations
+    with self.command_group('cosmosdb keys', cosmosdb_sdk) as g:
+        g.command('list', 'list_keys')
```

Comment:
why do you have `keys list` and `list-keys`? Presumably `list-keys` should be deprecated and hidden.

Reply:
I was adding the new command to be more consistent with other modules per https://github.com/Azure/azure-cli/issues/6094. Removing the existing command would introduce a breaking change and could lead to more open issues. I will put this change on hold for now.

Reply:
That's why you deprecate and hide the old command. It will continue to work, but will not be discoverable and will issue a warning. 
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments